packages/astro/test/streaming.test.js

```diff
@@ -48,7 +48,7 @@ describe('Streaming', () => {
 				let chunk = decoder.decode(bytes);
 				chunks.push(chunk);
 			}
-			expect(chunks.length).to.equal(3);
+			expect(chunks.length).to.equal(2);
```

Comment:
I _think_ because we render more stuff in sync now, it's now streamed in 2 chunks? This is the only test I had to tweak.